cdk/dcm_floodlight/mapping/track.yaml

```diff
@@ -0,0 +1,15 @@
+- destKey: dc_rdid
+  sourceKeys: context.device.advertisingId
+  required: true
+
+- destKey: ord
+  sourceKeys: properties.orderId
+  required: false
+
+- destKey: qty
+  sourceKeys: properties.quantity
```

Comment:
In other place it is also calculated from the sum of products array, anyway we can try that?